src/agent/driver-test-runner.js

```diff
@@ -127,7 +137,21 @@ export class DriverTestRunner {
           },
         });
       }
-    } else if (!capabilities.atName) {
+    } else if (!atName) {
+      return;
+    }
+    throw new Error(`Unable to ensure proper settings. Unknown atName ${atName}`);
+  }
+
+  /**
+   * Used for v1 aria-at tests, "reading" and "interaction" map to various settings based on AT.
+   * @param {"reading" | "interaction"} mode
+   */
+  async ensureMode(mode) {
+    const { atName } = await this.collectedCapabilities;
+    if (atName === 'NVDA') {
+      return this.ensureSettings(mode.toLowerCase() === 'reading' ? 'browseMode' : 'focusMode');
```

Comment:
```suggestion
      this.ensureSettings(mode.toLowerCase() === 'reading' ? 'browseMode' : 'focusMode');
      return;
```

Reply:
done (with the `await` that was missing from the suggestion)